build.sbt

```
@@ -1,8 +1,30 @@
 import scala.sys.process.Process
+import scala.io.Source
 
 import xerial.sbt.Sonatype._
 import Dependencies._
 
+// Load Sonatype Central credentials
+credentials += {
```

Need to report the parsing failure